src/service/url-replacements-impl.js

```diff
@@ -301,9 +301,12 @@ export class GlobalVariableSource extends VariableSource {
     this.setAsync('VARIANT', /** @type {AsyncResolverDef} */(experiment => {
       return this.getVariantsValue_(variants => {
         const variant = variants[/** @type {string} */(experiment)];
-        userAssert(variant !== undefined,
-            'The value passed to VARIANT() is not a valid experiment name:' +
-                experiment);
+        if (variant === undefined) {
+          user().error(TAG,
```

Comment:
Should this be happening here? Or should we replace `rethrowAsync` upstream with the `user().log()`? E.g. the logic that says "ignore errors and return empty string" is already part of this system. This duplicates it slightly.

Reply:
agree this is a duplicates. will need a root fix. was plan to have this as a temporary work around to unblock your PR

Reply:
will let your PR go if you can confirm the tests pass locally

Reply:
Yep. Can confirm that it works with patches.